browser/brave_profile_prefs_browsertest.cc

```diff
@@ -48,6 +53,12 @@ IN_PROC_BROWSER_TEST_F(BraveProfilePrefsBrowserTest, MiscBravePrefs) {
       browser()->profile()->GetPrefs()->GetBoolean(kIPFSCompanionEnabled));
   EXPECT_TRUE(
       browser()->profile()->GetPrefs()->GetBoolean(kBraveWalletEnabled));
+
+#if !BUILDFLAG(USE_GCM_FROM_PLATFORM)
+  EXPECT_FALSE(
+      browser()->profile()->GetPrefs()->GetBoolean(
```

Comment:
Is it possible to add any browser tests, checking that push messaging actually works? I.e. port them from Chromium?

Reply:
done!